README.md

```diff
@@ -39,6 +39,7 @@ git submodule update --init --recursive --depth 1
 To build the site in a container, run the following:
 
 ```bash
+make module-init
```

Comment:
```suggestion
# if you didn't yet run "git submodule update --init --recursive --depth 1",
# then this next command does that for you. You don't need to do this
# each time.
make module-init

```

People don't need to to this every time, and they only need to do that if they didn't follow the advice in the section immediately before this one.

Reply:
Understood will change it👍